pages/_document.js

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
```

Comment:
I would add a comment indicating this is the config from the MUI setup guide, in case someone is confused by all of this or the MUI config changes and we want to upgrade it